L/lwgrp/build_tarballs.jl

```diff
@@ -0,0 +1,90 @@
+# Note that this script can accept some limited command-line arguments,
+# run `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
+
+name = "lwgrp"
+version = v"1.0.6"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/LLNL/lwgrp", "7d8520d79d2a817fe3a8051b323183a8dd37ed2a"),
+    DirectorySource("bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/lwgrp
+
+export MPITRAMPOLINE_CC="${CC}"
+export MPITRAMPOLINE_CXX="${CXX}"
+export MPITRAMPOLINE_FC="${FC}"
+
+atomic_patch -p1 ../patches/libtoolize.patch
+atomic_patch -p1 ../patches/mpi.patch
+
+./autogen.sh
+mkdir build
+pushd build
+if [[ "${target}" == *-mingw* ]]; then
+    # There is no mpicc on Windows
+    ../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+else
+    ../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} CC=mpicc CXX=mpicxx
+fi
```

Comment:
Instead of duplicating function calls, only define variables conditionally (and `CC` and `CXX` already have default values)
```suggestion
if [[ "${target}" != *-mingw* ]]; then
    # There is no mpicc on Windows
    CC=mpicc
    CXX=mpicxx
fi
../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} CC=${CC} CXX=${CXX}
```
For what is worth, with autoconf you don't _have_ to build in a separate directory (which can still be cleaner though), because unlike CMake you have cleanup a build with `make distclean`.  CMake doesn't offer any way to restart a build from scratch.

Reply:
Done.